persistence/sql/persister_session.go

```diff
@@ -106,34 +107,27 @@ func (p *Persister) ListSessions(ctx context.Context, active *bool, paginatorOpt
 			return sqlcon.HandleError(err)
 		}
 
-		if expandables.Has(session.ExpandSessionIdentity) {
```

Comment:
@kelkarajay @jonas-jonas this would iterate over all sessions found, which is incredibly slow if you ahve large result sets. Instead, we're now using EagerPreload.

Reply:
Sounds good to me.